docs/intro/tutorial.rst

```diff
@@ -564,7 +564,7 @@ this time for scraping author information::
 
         def parse(self, response):
             # follow links to author pages
-            for href in response.css('.author a::attr(href)').extract():
+            for href in response.css('span a::attr(href)').extract():
```

Comment:
Great, that's cool, welcome and enjoy the journey! :)

Thanks for the change, can you update it to use `.author + ` instead of `span`?
I think you can do it directly here on Github, using the edit button (pencil icon at the right of the View button at the top of the file).